src/quantum/plugin/modifications/EnvironmentConditionModification.ts

```diff
@@ -9,7 +9,7 @@ export class EnvironmentConditionModification {
 
         return each(file.fuseboxIsEnvConditions, (replacable: ReplaceableBlock) => {
 
-            if (core.opts.isTargetUniveral()) {
+            if (core.opts.isTargetUniveral() || core.opts.isTargetNpm()) {
```

Comment:
I a not sure it's required. Target npm make a server bundle anyway. Once bundled an appropriate target will be set. So say, you publish with `npm` target but once bundled, you can choose `server` or `browser` or `universal`.

Reply:
"Target npm make a server bundle anyway"
That is where I disagree. I am trying to make a browser NPM package and it cannot deploy CSS because of that one line : it is considered as server while I am trying to make a browser-side npm package.

The produced NPM package produces this:
```
var runningInBrowser = false || 'npm' === 'electron';
```

while it should check for the presence of the `window` global variable instead of declaring it `false`. And this is what this PR is about.

I mean, I don't get where this idea that "npm = server" comes from. jQuery for exemple is on npm for G. sake, saying that jQuery is a server-side library and avoiding it to touch CSS is non-sense.

This PR brings no breaking, it just checks each time whether it is browser or server-side instead of blindly considering it is server-side. On the other hands, it allows browser-side npm packages to be bundled.